Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -328,25 +326,25 @@ bool CBoot::SetupWiiMemory(u64 ios_title_id)
 // Wii Bootstrap 2 HLE:
 // copy the apploader to 0x81200000
 // execute the apploader
-bool CBoot::EmulatedBS2_Wii()
+bool CBoot::EmulatedBS2_Wii(const DiscIO::IVolume* volume)
```

Comment:
I would prefer to keep it this way for now so that it's consistent with EmulatedBS2_GC, which does things we need even if no disc is inserted. In the future, we should look into splitting out code into a SetupGCMemory function so that we can call that instead of EmulatedBS2_GC when booting non-disc software so that we can make EmulatedBS2_GC expect valid volumes, but that's out of scope for this PR.